pkg/kubelet/container_gc.go

```diff
@@ -179,6 +204,11 @@ func (cgc *realContainerGC) removeOldestN(containers []containerGCInfo, toRemove
 		if err != nil {
 			glog.Warningf("Failed to remove dead container %q: %v", containers[i].name, err)
 		}
+		symlinkPath := path.Join(cgc.containerLogsDir, fmt.Sprintf("%s_%s-%s.log", containers[i].podName, containers[i].containerName, containers[i].id))
```

Comment:
Similar comment to above.  What keeps this in sync with the actual log file names?  It seems that you need a common utility function to map from container->logfilename.